docs/outputs.rst

```diff
@@ -22,207 +21,330 @@ FMRIPrep generates three broad classes of outcomes:
+(also known as *realignment*) of popular fMRI preprocessing software
+such as SPM_ or FSL_.
+One of the biggest advantages of *fMRIPrep* is the automatic calculation of multiple potential confounding variables
+beyond the standard head motion parameters.
+
+Confounding variables calculated in *fMRIPrep* are stored separately for each subject,
+session and run in :abbr:`TSV (tab-separated value)` files - one column for each confound variable.
+Such tabular files may include over 100 columns of potential confound regressors.
+
+.. warning::
+   Do not include all columns of `confounds_regressors.tsv` table
+   into your design matrix or denoising procedure.
+   Filter the table first, to include only the confounds you want to remove from your fMRI signal.
+   The choice of confounding variables may depend on the analysis you want to perform,
+   and may be not straightforward as no gold standard procedure exists.
+   For detailed description of various denoising strategies and their performance,
+   see [Parkes2018]_ and [Ciric2017]_.
+
+Confound regressors description
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Basic confouds
+==============
+
+- ``trans_x``, ``trans_y``, ``trans_z``, ``rot_x``, ``rot_y``, ``rot_z`` - the 6 rigid-body motion
+  parameters (3 translations and 3 rotation) estimated relative to a reference image;
+
+- ``csf`` - the average signal within anatomically-derived eroded :abbr:`CSF (cerebro-spinal fluid)` mask;
+
+- ``white_matter`` - the average signal within  the anatomically-derived eroded :abbr:`WM (white matter)` masks;
+
+- ``global_signal`` -  the average signal within the brain mask.
+
+Parameter expansion of basic confounds
+======================================
+Only accounting for the standard six motion parameters may not be sufficient to remove all variance related to
+head motion from the fMRI signal.
+Thus, Friston et al. [Friston1996]_ and Satterthwaite et al. [Satterthwaite2013]_
+proposed *24-motion-parameter* expansion, with a goal of removing from fMRI signal as much of the motion-related
+variance as possible.
+To make this technique more accessible, *fMRIPrep* automaticaly calculates motion parameter
+expansion [Satterthwaite2013]_, providing timeseries corresponding to first *temporal derivatives* of six motion
+parameters, together with their *quadratic terms*, resulting in the total 24 head motion parameters
+(6 standard motion parameters + 6 temporal derivatives of six motion parameters + 12 quadratic terms of 6 motion
+parameters and their 6 temporal derivatives).
+Additionally, *fMRIPrep* returns temporal derivatives and quadratic terms for the ``csf``, ``white_matter``
+and ``global_signal`` to enable applying 36-parameter denoising strategy
+proposed by Satterthwaite et al. [Satterthwaite2013]_.
+
+Derivatives and quadratic terms are stored under column names with suffixes: ``_derivative1`` and powers ``_power2``.
```

Comment:
We should probably note that these are not BIDS-Derivatives-Draft conformant and subject to change in future versions.